virttest/qemu_devices/qcontainer.py

```diff
@@ -2279,19 +2279,18 @@ def pcic_by_params(self, name, params):
                 name, bus_type, 'xio3130-downstream', name)
             driver = 'x3130-upstream'
         else:
+            bus_length = 23
```

Comment:
Would you please explain this line? doesn't it be `32`?

Reply:
oh, it's a typo. let's fix it